python/vowpalwabbit/pyvw.py

```diff
@@ -597,11 +601,15 @@ def __init__(self, costs=[], prediction=0):
     def from_example(self, ex):
         class wclass:
             def __init__(self,
-                         label,
-                         cost=0.,
-                         partial_prediction=0.,
-                         probability=0.):
-                self.label = label
+                        action=None,
+                        cost=0.,
+                        partial_prediction=0.,
+                        probability=0.,
+                        **kwargs):
+                if kwargs.get('label', False):
+                    action = kwargs['label']
+                    warnings.warn("label has been deprecated. Please use 'action' instead.", DeprecationWarning)
```

Comment:
Deprecating should not break existing functionality. If someone tries to retrieve the `label` field it will fail now

Reply:
Okay, I will fix it now!